Plugins~/Src/MeshSyncClientBlender/BlenderPyObjects/BlenderPyDepsgraph.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <BKE_context.h> //bContext
+#include "MeshUtils/muMath.h"
+
+namespace blender
+{
+
+    class BlenderPyDepsgraph
+    {
+    public:
+        BlenderPyDepsgraph(Depsgraph* depsgraph);
```

Comment:
Documentation needed

Reply:
Added XML comments on class and functions